src/app/services/transmart-resource.service.ts

```diff
@@ -520,4 +520,30 @@ export class TransmartResourceService {
       }));
   }
 
+  /**
+   * Gets an icon per subject dimension name
+   * TODO TMT-773 - make dimension icons configurable
+   * @param dimensionName
+   */
+  getSubjectDimensionIcon(dimensionName: string): string {
```

Comment:
This does not seem to be the right place. Please create a separat class for this.